tests/network/vmi_lifecycle.go

```diff
@@ -128,3 +129,13 @@ var _ = SIGDescribe("[crit:high][arm64][vendor:cnv-qe@redhat.com][level:componen
 func renderPkillAllPod(processName string) *k8sv1.Pod {
 	return tests.RenderPrivilegedPod("vmi-killer", []string{"pkill"}, []string{"-9", processName})
 }
+
+func VerifyDummyNicForBridgeNetwork(vmi *v1.VirtualMachineInstance) {
```

Comment:
This function can be private, because it is used only in this file.

Reply:
Done